eng/common/scripts/create-tags-and-git-release.ps1

```diff
@@ -6,11 +6,8 @@ param (
   # used by VerifyPackages
   $artifactLocation, # the root of the artifact folder. DevOps $(System.ArtifactsDirectory)
   $workingDirectory, # directory that package artifacts will be extracted into for examination (if necessary)
-  $packageRepository, # used to indicate destination against which we will check the existing version.
```

Comment:
To enable flexibility perhaps we just default this value from the global instead of removing it as a parameter.